### DISCUSSION TITLE
On screenkeyboard.

### DISCUSSION THREAD

**Discussion by Kendarr443** [LizardByte/community#79] [Sunshine]
Hello, is it possible to add an onscreen keyboard? Like the one present in Steam link, maybe holding the select button, many times I need to type something quick, but I need to run up to my machine. Sorry if this is a weird post, I never used this feature in GitHub.

Thanks for the amazing project, it is way more stable than Steam Link and allows me to have console like experience in my living room :)

This is roughly how it looks. 
![image](https://user-images.githubusercontent.com/80401945/190000442-17f81fcf-c29c-4bbf-9b01-ae155e74053e.png)

**Comment by ReenigneArcher** [comment]
This is a good idea! Nvidia GameStream also has one if you use their Nvidia Shield app instead of Moonlight.

**Reply by Kendarr443** [reply]
I haven't tried that app to be honest, will try thanks!.

**Reply by ReenigneArcher** [reply]
To be clear, Nvidia GameStream won't work with Sunshine. I was just pointing out another example.

**Reply by Kendarr443** [reply]
I could not find it, do you mean the actual nvidia tv product?, I have a Xiaomi Mistick it's where I use moonlight to connect and play, it's android.

**Reply by ReenigneArcher** [reply]
Yes, Nvidia Gamestream on Nvidia Shield TV device, connected to GeForce Experience.

**Reply by Kendarr443** [reply]
Ah, alright!

**Reply by skarabaraks** [reply]
Amazing suggestion. On screen keyboard similar to Gamestream would definitely strengthen + add value to sunshine

**Reply by mallah94** [reply]
hey maybe very buggy, but for now i am using this as a small workaround [https://github.com/mallah94/OnScreenXboxKeyboard](url)  it is very hacky and chopped together.

**Reply by ld892012** [reply]
There is also [JoyXoff ](https://joyxoff.com). This software have a virtual keyboard and can pass UAC prompts ^^
